examples/cxx/health_monitoring/src/central_daemon.cpp

```diff
@@ -31,7 +31,7 @@ auto main() -> int {
     auto service_name_2 = ServiceName::create("service_2").expect("");
 
     auto node = NodeBuilder()
-                    .name(NodeName::create("central daemon").expect(""))
+                    .name(NodeName::create("central daemon").value())
```

Comment:
@ComicSansMS @elfenpiff @FerdinandSpitzschnueffler with the minimal STL compatible `Expected`, we lose the functional interface, similar to the new `Optional`. The question is now how to handle this cases where we previously used just `expect("foo")`?

We can just call `value` and with our implementation we can just abort but other implementations might trigger UB. In this simple case, we could just do the proper error handling, but there are other cases where this becomes quite cumbersome.

An idea would be to have a helper function like this
```
template <typename T, typename E>
auto unwrap_or_panic(Expected<T, E> exp) -> T;
```

What do you think? Ideas welcome.